src/integrations/pact-axios/pact-axios.ts

```diff
@@ -81,4 +87,53 @@ export class PactAxios {
             .find((argument) => argument.getType().getProperties()[0]?.getEscapedName() === property);
         return axiosConfig?.getType().getProperty(property)?.getValueDeclaration()?.getType();
     }
+
+    /** Recursively find path under axios url argument - which might be string, template string, concatenation, variable etc. */
+    private generatePathFromNode(node: tsMorph.Node): string {
+        // simple string
+        if (node.getType().isStringLiteral()) {
+            return node.getType().getText().replace(/["']/g, '');
```

Comment:
`'` is a valid URL character, we cannot delete them here
src: https://stackoverflow.com/questions/1547899/which-characters-make-a-url-invalid/1547940#1547940